### DISCUSSION TITLE
Building keycloak with a docker

### DISCUSSION THREAD

**Discussion by tzahifadida** [keycloak/keycloak#15745] [Ideas]
I wish to build the docker image like in the official distribution for arch linux/amd64 (from my arm machine) (https://quay.io/repository/keycloak/keycloak).
I found the Dockerfile in quarkus/container directory.
Is this considered a similar production grade dockerfile?
I.e. security wise?
Is this the same Dockerfile that builds https://quay.io/repository/keycloak/keycloak

Thanks.

**Comment by sschu** [comment]
AFAIK yes, this is the same.

**Reply by tzahifadida** [reply]
Thanks!
I really wanted to try this improvement - https://github.com/keycloak/keycloak/issues/14113

Works very nice. Yay!
